notebooks/00020_fix_positions_hg38_missense_pathogenic.ipynb

```diff
@@ -1224,7 +1224,11 @@
    "id": "162968cb-3f83-4aab-8c76-e557e5a21cc0",
```

Comment:
<p>@dianagroch can you please check this?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/zdenkas/human-tafazzin-variants-database/pull/16/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>

Reply:
<h4>hg38 X:154412187 - is three time in dataset (rows 57, 58 and 65)</h4><h4><br></h4><p>for rows 57 and 58:</p><h4>in position hg38 X:154412187 is T (checked at UCSC and NCBI)</h4><p>so &nbsp;&nbsp;c.211T&gt;C should be correct </p><p> </p><p>for row 65: </p><p>does not fit conversion from hg37 should be hg38 <a href="https://genome.ucsc.edu/cgi-bin/hgTracks?db=hg38&amp;position=chrX:154412202-154412205" target="_blank">chrX:154412203</a> (not 154412187)</p><p>should be C in this position</p><p>than this is correct: NM_000116.5(TAFAZZIN):c.227C&gt;G (p.Pro76Arg)</p><p> </p><p> </p><p> </p><p> </p><p> </p>